sdk/src/types/block/slot/epoch.rs

```diff
@@ -57,22 +57,43 @@ pub struct EpochIndex(pub u32);
 impl EpochIndex {
     /// Gets the range of slots this epoch contains.
     pub fn slot_index_range(&self, slots_per_epoch_exponent: u8) -> core::ops::RangeInclusive<SlotIndex> {
-        self.first_slot_index(slots_per_epoch_exponent)..=self.last_slot_index(slots_per_epoch_exponent)
+        // Genesis slot hardcoded to 0 since it doesn't affect the range.
```

Comment:
How does it not affect the range ?

Reply:
Somehow I only thought about the range size, not about the actual values of it :joy: Fixed